.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 orbs:
   android-sdk: rakutentech/android-sdk@0.2.1
```

Comment:
@khairul-alam-licon @Climbatize  does this orb needs to be updated to cache the mapping file in circle ci? or its already added?

Reply:
That's a good point. I think it should work because it's already persisting the whole `output` folder to the workspace: https://github.com/rakutentech/android-buildconfig/blob/master/circleci/android-sdk/config.yml#L79

But I just noticed it looks like the path being passed to the mapping job is incorrect.